tools/spec2vec/spec2vec_training_wrapper.py

```diff
@@ -0,0 +1,108 @@
+import argparse
```

Comment:
A shebang would be nice here `#!/usr/bin/env python`

Reply:
Fixed in 8ec040a.